src/runtime_src/driver/zynq/drm/zocl/zocl_bo.c

```diff
@@ -276,15 +276,19 @@ zocl_userptr_bo_ioctl(struct drm_device *dev, void *data, struct drm_file *filp)
 	struct page **pages;
 	unsigned int sg_count;
 
-	if (offset_in_page(args->addr))
+	if (offset_in_page(args->addr)) {
+		DRM_ERROR("User ptr not PAGE aligned\n");
 		return -EINVAL;
+	}
 
-	if (args->flags & XCL_BO_FLAGS_EXECBUF)
+	if (args->flags & XCL_BO_FLAGS_EXECBUF) {
+		DRM_ERROR("Exec buf could not be a user buffer\n");
 		return -EINVAL;
+	}
 
 	bo = zocl_create_userprt_bo(dev, args->size);
 	if (IS_ERR(bo)) {
-		DRM_DEBUG("object creation failed\n");
+		DRM_INFO("Object creation failed\n");
```

Comment:
Why DRM_INFO is used here instead of DRM_ERROR?

Reply:
Yes. DDR_ERROR should be used.